packages/core-extensions/content-containers/exam-edit/src/edit/AssessmentGroup.vue

```diff
@@ -0,0 +1,238 @@
+            @delete="deleteAssessment(element)" />
+        </template>
+      </ElementList>
+    </VExpansionPanelText>
+  </VExpansionPanel>
+</template>
+
+<script lang="ts" setup>
+import { computed, inject, ref, watch } from 'vue';
+import { numberToLetter, uuid } from '@tailor-cms/utils';
+import type { Activity } from '@tailor-cms/interfaces/activity';
+import cloneDeep from 'lodash/cloneDeep';
+import type { ContentElement } from '@tailor-cms/interfaces/content-element';
+import debounce from 'lodash/debounce';
+import { ElementList } from '@tailor-cms/core-components';
+import type { ElementRegistry } from '@tailor-cms/interfaces/schema';
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import map from 'lodash/map';
+import pickBy from 'lodash/pickBy';
+import sortBy from 'lodash/sortBy';
+import uniq from 'lodash/uniq';
+
+import AssessmentItem from './Assessment.vue';
+import GroupIntroduction from './GroupIntroduction.vue';
+
+interface Props {
+  group: Activity;
+  elements: Record<string, ContentElement>;
+  objectives: Activity[];
+  position: number;
+  isDisabled?: boolean;
+  isExpanded?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  isDisabled: false,
+  isExpanded: false,
+});
+
+const emit = defineEmits([
+  'update',
+  'delete',
+  'save:element',
+  'update:element',
+  'reorder:element',
+  'delete:element',
+]);
+
+const schemaService = inject<any>('$schemaService');
+const ceRegistry = inject<ElementRegistry>('$ceRegistry');
+
+const unsavedAssessments = ref<Record<string, any>>({});
+const timeLimit = ref<number>(get(props.group, 'data.timeLimit', 0));
+
+const questionTypes = computed(() => map(ceRegistry?.questions, 'type'));
+const savedAssessments = computed(() => {
+  const filtered = filter(props.elements, (el) => {
+    return el.activityId === props.group.id && isQuestion(el.type);
+  });
+  return sortBy(filtered, 'position');
+});
+
+const introductionElements = computed(() => {
+  const cond = (it: any) => it.activityId === props.group.id && !isQuestion(it.type);
+  return sortBy(filter(props.elements, cond), 'position');
+});
+
+const assessments = computed(() => {
+  return sortBy([
+    ...savedAssessments.value,
+    ...Object.values(unsavedAssessments.value),
+  ], 'position');
+});
+
+const hasAssessments = computed(() => {
+  return !isEmpty(assessments.value);
+});
+
+const objectiveLabel = computed(() => {
+  if (isEmpty(props.objectives)) return '';
+  const types = uniq(map(props.objectives, 'type'));
+  const label = types.length > 1
+    ? 'Objective'
+    : schemaService.getLevel(types[0]).label;
+  return `Link ${label}`;
+});
+
+const supportedElementConfig = computed(() => {
+  const items = questionTypes.value.map((id) => ({ id, isGradable: true }));
+  return [{ name: 'Assessments', items }];
+});
+
+const isQuestion = (type: string) => questionTypes.value.includes(type);
```

Comment:
Do we have this within schemaService / if not should we ?

Reply:
`schemaService` doesn't have access to the registry, but I added `isQuestion` helper to `ceRegistry`